src/Xlsx.php

```diff
@@ -524,4 +524,14 @@ private function replaceValue($value)
             return false;
         }
     }
+
+    /**
+    *duplicateStyle
+    */
+    public function duplicateStyle($cellStyle = null, $range = '')
+    {
+        $sheet = $this->xlsx->getActiveSheet();
+
+        $sheet->duplicateStyle($sheet->getStyle($cellStyle), $range);
```

Comment:
## Point1. Usecase: Copy style from sheet A's cell to sheet B's cell.

If we want to copy style from sheet A's cell to sheet B's cell, this method can not.

For example, how about `deplicateStyle($fromOption, $toOption)` ?